src/request/node.ts

```diff
@@ -15,6 +15,8 @@ import { generateRequestUrl } from './util';
 // TODO: Where should the dojo version come from? It used to be kernel, but we don't have that.
 let version = '2.0.0-pre';
 
+const defaultRedirectLimit = 15;
```

Comment:
We should UPPER_CASE `const` values that we use as *constants* like this.  So `DEFAULT_REDIRECT_LIMIT`